pagopa/io-app
# chore: [IOCOM-2001] Revert label content for empty FIMS history

## Short description
This PR reverts the label's content for the empty FIMS history.
![Simulator Screenshot - iPhone 16 - 2024-12-04 at 12 33 37](https://github.com/user-attachments/assets/c08221b7-9537-4685-ba6a-25f256c58542)

## List of changes proposed in this pull request
- Label content reverted

## How to test
Using the io-dev-api-server, set 0 items for the FIMS' history and check the related screen.
